isolated-contexts.bs

```diff
@@ -57,47 +57,137 @@ urlPrefix: https://w3c.github.io/webappsec-csp/; spec:CSP3
 
 # Introduction # {#introduction}
 
-Many powerful capabilities cannot safely be enabled in the Web Platform as it
-is defined today. Some may break the platform's fundamental security
-primitives, others may be too complex or technical to expect users to make
-informed decisions about. Exposing these capabilities requires establishing
-more trust in a page's content than the web can currently provide.
-[Isolated Web Apps](https://github.com/WICG/isolated-web-apps) define one method
-of establishing this trust by requiring third-party attestation before
-enabling particularly powerful capabilities.
+Throughout its existence the web has been evolved into an increasingly capable
+application platform. Secure Contexts formalize transport security, Cross
+Origin Isolation mitigates side-channel attacks, and device-specific permission
+prompts make device access more scoped and understandable to users. Each of
```

Comment:
I would say something like "browsers and operating systems have experimented with permission interfaces based on selecting particular files and devices, to make access more scoped[...]".